src/fcu_io.cpp

```diff
@@ -302,6 +320,8 @@ void fcuIO::handle_small_imu_msg(const mavlink_message_t &msg)
                             &imu_msg.angular_velocity.z,
                             &temp_msg.temperature);
 
+  imu_msg.orientation = attitude_quat_;
```

Comment:
I'm wondering if we want to do this? The orientation estimate isn't really produced by the IMU per se, and it seems like since we have a dedicated attitude message for the output of the complimentary filter, duplicating it here might be more confusing than its worth since it seems like something separate when it's really the same. Do think it'd be better to just leave this unpopulated?